vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -46,10 +46,21 @@
     shell: "kustomize build /home/$USER/go/src/github.com/metal3-io/baremetal-operator/deploy/default/ | kubectl --kubeconfig=/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml apply -n {{ NAMESPACE }} -f -"
     when: CAPM3_VERSION == "v1alpha3"
 
+  - name: Ironic TLS setup and Ironic basic authentification sourcing
+    shell: "source ./{{ metal_dir }}/lib/{{ item }}"
+    loop: 
+      - ironic_tls_setup.sh
+      - ironic_basic_auth.sh
+
   # Install Ironic
   - name: Install Ironic
-    shell: "kustomize build /home/$USER/go/src/github.com/metal3-io/baremetal-operator/ironic-deployment/keepalived/ | kubectl --kubeconfig=/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml apply -n {{ NAMESPACE }} -f -"
-
+    shell: "{{ BMOPATH }}/tools/deploy.sh"
+    args:
```

Comment:
the shell module has no args afaik

Reply:
I think it has. But apparently it has a different purpose, not for passing the parameters to the script.